vxfreeswitch/voice.py

```diff
@@ -474,7 +476,7 @@ def send_outbound_message(self, client, message):
     def client_answered(self, client):
         """Function that is called when the ChannelAnswer event is received.
         Fires the deferred related to the outbound call"""
-        d = self._unanswered_channels.pop(client.get_address(), None)
+        d = self._unanswered_channels.get(client.get_address(), None)
```

Comment:
What happens in the callback that requires the client address to still be in the unanswered channels list?